scenarios/tracking/FAQ.md

```diff
@@ -0,0 +1,173 @@
+# Multi-Object Tracking
+
+```diff
++ June 2020: This work is ongoing.
+```
+
+## Frequently asked questions
+
+This document tries to answer frequent questions related to multi-object tracking. For generic Machine Learning questions, such as "How many training examples do I need?" or "How to monitor GPU usage during training?" see also the image classification [FAQ](https://github.com/microsoft/ComputerVision/blob/master/classification/FAQ.md).
```

Comment:
This document covers content regarding common questions and topics related to multi-object tracking.

I think this sounds better for the first line? I don't like the 'tries' word choice.